torch/csrc/jit/codegen/cuda/executor_utils.cpp

```diff
@@ -1050,26 +1058,22 @@ NvrtcFunction nvrtcCompile(
     const auto result = at::globalContext().getNVRTC().nvrtcCompileProgram(
         program, args.size(), args.data());
 
-    if (result != NVRTC_SUCCESS) {
-      // NOLINTNEXTLINE(cppcoreguidelines-init-variables)
-      size_t logsize;
-      at::globalContext().getNVRTC().nvrtcGetProgramLogSize(program, &logsize);
-      // NOLINTNEXTLINE(cppcoreguidelines-init-variables)
-      std::vector<char> log(logsize);
-      at::globalContext().getNVRTC().nvrtcGetProgramLog(program, log.data());
+    // NOLINTNEXTLINE(cppcoreguidelines-init-variables)
+    size_t logsize;
+    at::globalContext().getNVRTC().nvrtcGetProgramLogSize(program, &logsize);
+    // NOLINTNEXTLINE(cppcoreguidelines-init-variables)
+    std::vector<char> log(logsize);
+    at::globalContext().getNVRTC().nvrtcGetProgramLog(program, log.data());
```

Comment:
Not sure if there's significant overhead... But looks like we are always extracting log now.

Reply:
I doubt it is relative to compiling with nvrtc.